src/main/java/org/example/C05_04_Copy_Array.java

```diff
@@ -0,0 +1,26 @@
+package org.example;
+
+public class C05_04_Copy_Array {
+    public static void main(String[] args) {
+        int[] elements1 = new int[]{1, 9, 8, 9};
+        //Create another array arr2 with size of arr1
```

Comment:
Comment is not correct, there is no arr2 or arr1. You better remove the comment altogether.

Reply:
done